README.md

```diff
@@ -0,0 +1,37 @@
+This is a simple TODO app built with using **[JavaScript in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/javascript/javascript-in-visual-studio?view=vs-2022)**.
+This front-end uses React and Vite.
+
+This is a demo project showing how you can create React apps in Visual Studio. The todo items are stored in memory in this sample app.
+
+## Get started in Visual Studio 2022
+To get started with this in Visual Studio follow the steps below.
+
+- Install Visual Studio 2022 with the Node.js workload as well as the Web workload
+- Clone this repo locally
+- F5 to Start Debugging
+
+## Features
+
+- **Add Tasks**: Users can input and add new tasks to the list.
+- **Move Tasks**: Tasks can be moved up and down in the list.
+- **Delete Tasks**: Users can remove tasks from the list.
+- **Interactive UI**: Buttons allow task management directly from the interface.
+
+## Get started with the CLI
+
+Ensure that you have the following components installed.
+
+- nodejs
+- npm
+
+To run the app execute the commands below.
+
+```bash
+> cd todojswebapp
+> npm install
```

Comment:
Since the project contains a lock-file, `npm ci` it's better here.

Reply:
Thanks, I'll update that.